src/components/Navigation/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Styled from "./styled";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faBars, faUser } from '@fortawesome/free-solid-svg-icons'
+
+function Navigation() {
+  return <Styled.Navigation>
```

Comment:
For good practice and standards sake:

It is a common practice to use parenthesis when returning a complete node of elements:

return (
   <Styled.Navigation>
      ...
   </Styled.Navigation>
);